protos/service.proto

```diff
@@ -0,0 +1,86 @@
+syntax = "proto3";
+
+package tasktimer;
+
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/nivista/tasktimer/.gen;pb";
+
+service TaskTimer {
+    rpc CreateTimer (CreateTimerRequest) returns (CreateTimerResponse) {}
+
+    rpc DeleteTimer (DeleteTimerRequest) returns (DeleteTimerResponse) {}
+
+    rpc GetTimer (GetTimerRequest) returns (GetTimerResponse) {}
+}
+
+message CreateTimerRequest {
+    string account = 1;
+    TaskConfig task_config = 2;
+    ScheduleConfig schedule_config = 3;
+}
+
+message CreateTimerResponse {
+    string id = 1;
+}
+
+message DeleteTimerRequest {
+    string account = 1;
+    string id = 2;
+}
+
+message DeleteTimerResponse {}
+
+message GetTimerRequest {
+    string account = 1;
+    string id = 2;
+}
+
+message GetTimerResponse {
+    bytes id = 1;
+    uint32 execution_count = 2;
```

Comment:
I think the GetTimerResponse struct should look as follows 

```
string id = 1;
Progress progress = 2;
Metadata metadata = 3;
CreateRequest createRequest = 4;
```

Reply:
ok interesting idea.